plugin/trino-delta-lake/src/test/resources/databricks133/v2_checkpoint_json/_delta_log/00000000000000000001.checkpoint.ae4ea00c-afe0-4e8b-ad87-28125262bd44.json

```diff
@@ -0,0 +1,4 @@
+{"sidecar":{"path":"00000000000000000001.checkpoint.0000000001.0000000001.59e66709-d0ca-402d-9375-0e3c4fcf8d73.parquet","sizeInBytes":12007,"modificationTime":1708995845000}}
```

Comment:
it would be nice to have a resource to test reading from a checkpoint with multiple sidecars.
i haven't found any knobs to do this - just found in delta.io code unit tests for writing such checkpoints.
in anycase, optional at this point.